src/urg_stamped.cpp

```diff
@@ -433,15 +450,33 @@ class UrgStampedNode
     scip_->sendCommand("TM0");
   }
 
-  void errorCountIncrement()
+  void errorCountIncrement(const std::string& status)
   {
-    ++error_count_;
-    if (error_count_ > error_count_max_)
+    if (status == "00")
+      return;
+
+    if (status == "0L")
     {
-      ROS_ERROR("Error count exceeded limit, resetting the sensor and exiting.");
-      scip_->sendCommand("RS");
-      ros::Duration(0.05).sleep();
-      ros::shutdown();
+      ++error_count_.abnormal_error;
+      if (error_count_.abnormal_error > error_count_max_)
+      {
+        ROS_ERROR("Error count exceeded limit, trying to reboot the sensor and exiting.");
+        scip_->sendCommand("RB");
+        scip_->sendCommand("RB");  // Sending it 2 times in 1 sec. is needed
+        ros::Duration(0.05).sleep();
+        ros::shutdown();
```

Comment:
It may shutdown the node before handling `cbRB` response.

Reply:
After testing using LIDAR, it sometimes does not send response from `RB` command.
Thus I wrote `ros::shutdown();` here, instead of in `cbRB`.

I ensured just adding service server that callback is simply the following.

```
  bool cbReboot(std_srvs::TriggerRequest& req, std_srvs::TriggerResponse& res)
  {
    ROS_INFO("cbReboot called");
    scip_->sendCommand("RB");
    return true;
  }
```

However, the LIDAR always reboots. So shutting down the node at L467 before waiting `cbRB` is no problem, I think.